jupyter-on-gke/README.md

```diff
@@ -36,13 +36,20 @@ Preinstall the following on your computer:
 
 3. Once the image is built, navigate to `ai-on-gke/jupyter-on-gke/`
 
-4. Edit `variables.tf` with your GCP settings. The `<your user name>` that you specify will become a K8s namespace for your Jupyterhub services.
+4. Edit `variables.tf` with your GCP settings. The `<your user name>` that you specify will become a K8s namespace for your Jupyterhub services. For more information about what the variabls do visit [here](https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/jupyter-on-gke/variable_definitions.md)
 **Important Note:**
 If using this with the Ray module (`ai-on-gke/ray-on-gke/`), it is recommended to use the same k8s namespace
 for both i.e. set this to the same namespace as `ai-on-gke/ray-on-gke/user/variables.tf`.
 If not, set `enable_create_namespace` to `true` so a new k8s namespace is created for the Jupyter resources.
-**Note:**
-We allow user to set their own domains, in the `variables.tf` file. Since we are also using an Ingress Object, it is required for the Ingress to also have specifiy the name of the global static address.
+
+5. If you have not enabled the IAP API before or created a Brand for your project, you can proceed to the next step. If not, ensure that the following variables within `variables.tf` are set:
```

Comment:
How would a customer know how to check this? They may have enabled it a year ago and forgotten. Do they get an error if they have enabled it and don't set the fields below, or does logging in just not work?

Can you add some links to Pantheon where they can check if they already have IAP/brand or describe steps to check?

Reply:
So terraform will fail if it is already enabled. But I will add links to IAP and pictures of what it would look like.

Reply:
I've added screenshots of what it would look like if there is not brand and IAP isn't enabled.